src/QMCWaveFunctions/Fermion/MatrixUpdateOMPTarget.h

```diff
@@ -302,8 +302,7 @@ class MatrixUpdateOMPTarget
     Value* Ainv_ptr       = Ainv.data();
     Value* temp_ptr       = temp.data();
     Value* rcopy_ptr      = rcopy.data();
-    PRAGMA_OFFLOAD("omp target data map(always, to: phiV_ptr[:norb]) \
```

Comment:
The H2D copy has been moved to the caller and honor `phiV_ptr` as a const ptr.

Reply:
Ok better to have this explicit instead of hiding here in the pragma.